### PR TITLE
added delete all as well as add all repositories under chosen root. U…

### PR DESCRIPTION
**Pull request by franchert**
…pdated documentation to reflect changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,15 @@ underscores is hard. So for example, `add_repo` can be typed as `addrepo`.
                                 name given in the second parameter. The location
                                 of the repo will be asked for and then added to
                                 your list of repos.
+* `repo_add_directory` or `rad` - Searches all subdirectories of the second parameter
+                                  and automatically adds each to the list of repos. This
```
Comment (iamsellek) [inline README.md:72]: Doesn't it only add subdirectories with a .git directory in them? If that is the case, this should be updated to reflect that. If it isn't the case, I think it should be.

File: README.md [hunk 2]
```diff
@@ -68,8 +68,15 @@ underscores is hard. So for example, `add_repo` can be typed as `addrepo`.
...
+                                  command uses the relative path between the second parameter 
+                                  and the repo as the repo name. The second parameter can be
```
Comment (iamsellek) [inline README.md:74]: I'd like to change this to have it name the repo as just the name of the directory with the .git directory in it. For instance, if our second parameter is a directory named "code" and we have a directory that is a repo inside of that called "website", the repo name in alfred would just be "website."

Comment (franchert) [reply README.md:74]: Only issue I can see with that is what happens when we have duplicates? There were a couple  of generic repo names that would have duplicate entries if you ran it on the dudewheresmycode output folder. For example, the "dsi/root" and "dsi_api/root".

Maybe upon finding a duplicate it does the full path for the second item and beyond?

File: README.md (hunk 1 above)
Comment (franchert) [reply README.md:72]: You're right, it's a bit ambiguous on waht it adds. I'm going to reword it to make it clear.
